Assets/MixedRealityToolkit.SDK/Experimental/Features/Utilities/TapToPlace/TapToPlace.cs

```diff
@@ -0,0 +1,467 @@
+            "The GameObjectToPlace will be placed at the default placement distance if a surface is not hit by the raycast.")]
+        private float defaultPlacementDistance = 1.5f;
+
+        /// <summary>
+        /// The default distance (in meters) an object will be placed relative to the TrackedTargetType forward in the SolverHandler.
+        /// The GameObjectToPlace will be placed at the default placement distance if a surface is not hit by the raycast.
+        /// </summary>
+        public float DefaultPlacementDistance
+        {
+            get => defaultPlacementDistance;
+            set
+            {
+                if (defaultPlacementDistance != value)
+                {
+                    defaultPlacementDistance = value;
+                }
+            }
+        }
+
+        [SerializeField]
+        [Tooltip("Max distance to place an object if there is a raycast hit on a surface")]
+        private float maxRaycastDistance = 20.0f;
+
+        /// <summary>
+        /// The max distance to place an object if there is a raycast hit on a surface
+        /// </summary>
+        public float MaxRaycastDistance
+        {
+            get => maxRaycastDistance;
+            set => maxRaycastDistance = value;
+        }
+
+        [SerializeField]
+        [Tooltip("Array of LayerMask to execute from highest to lowest priority. First layermask to provide a raycast hit will be used by component")]
+        private LayerMask[] magneticSurfaces = { UnityEngine.Physics.DefaultRaycastLayers };
+
+        /// <summary>
+        /// Array of LayerMask to execute from highest to lowest priority. First layermask to provide a raycast hit will be used by component.
+        /// </summary>
+        public LayerMask[] MagneticSurfaces
+        {
+            get => magneticSurfaces;
+            set => magneticSurfaces = value;
+        }
+
+        //[SerializeField]
+        //[Tooltip("Is the gameObjectToPlace currently in a state where it is being placed? This state is activated when you selecthe GameObjectToPlace.")]
+        //private bool isBeingPlaced = false;
+
+        /// <summary>
+        /// Is the gameObjectToPlace currently in a state where it is being placed? This state is activated when you select
+        /// the GameObjectToPlace.
+        /// </summary>
+        public bool IsBeingPlaced { get; protected set; }
+
+        [SerializeField]
+        [Tooltip("If true, the gameObjectToPlace will start in the placing state.  Meaning that the object will immediately start" +
+        "by following the TrackedTargetType (Head or Controller Ray) and then a tap is required to place the object.  AutoStart" +
+         "is only called once because it is in Start();")]
+        private bool autoStart = false;
+
+        /// <summary>
+        /// If true, the gameObjectToPlace will start in the placing state.  Meaning that the object will immediately start
```

Comment:
Same comments on these comments as well (I think we'll fix up the on above and bring those changes down here.

Reply:
fixed